openbas-api/src/main/java/io/openbas/rest/tag/form/TagUpdateInput.java

```diff
@@ -3,31 +3,20 @@
 import static io.openbas.config.AppConfig.MANDATORY_MESSAGE;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
 import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
 
+@Data
```

Comment:
Maybe not relevant to this PR, but we should probably create a super class TagInput for update, create, etc. There are some fields in common

Reply:
Doing like for the team